docs/redoc/master/openapi.json

```diff
@@ -10984,6 +10984,7 @@
         "type": "object",
         "required": [
           "config",
+          "hardware_usage",
```

Comment:
Shouldn't this be optional?

Reply:
This is the telemetry endpoint and as far as I understood, we wanted it to be able to turn off for each API that gets measured (like search or matrix), but not telemetry.
I could be wrong here though.

Maybe @generall can give his opinion, whether we want it always to be in telemetry endpoint, or only in a certain condition.

Reply:
It should be option at least for backward compatibility of the clients. If new client reads old telemetry output it should not fail